hydrus/tests/conftest.py

```diff
@@ -46,7 +48,16 @@ def gen_dummy_object(class_title, doc):
     expanded_base_url = DocUrl.doc_url
     for class_path in doc.collections:
         if class_title == doc.collections[class_path]["collection"].name:
-            members = [str(uuid.uuid4()) for _ in range(3)]
+            members = list()
+            for _ in range(3):
```

Comment:
This test also needs to be modified. We should only create dummy objects for classes in the manages block of the collection. For any other type it should return an error. Add a separate test for the error case.

Reply:
Okay. Understood. Will make the change